routes/dfaf/user.json

```diff
@@ -0,0 +1 @@
+{"name":"Dima","age":20}
```

Comment:
Please remove this file. It's not secret, but it's not normal safe user data

Reply:
done

Reply:
you have not removed this file